src/block-shapes.ts

```diff
@@ -0,0 +1,329 @@
+enum BlockShapes {
+    HAT = 'hat',
+    BOOLEAN = 'boolean',
+    REPORTER = 'reporter',
+    C = 'c',
+    CAP = 'cap',
+    STACK = 'stack',
+    CUSTOM = 'custom',
+}
+
+enum BlockOpcodes {
```

Comment:
Are enums automatically exported? I think it would be useful to exports these enums, maybe from their own file.

Reply:
I want to make these "data" bits loaded eventually, some thoughts about that in the future of the lib i think